packages/event_store/src/storage.js

```diff
@@ -114,6 +143,22 @@ class Storage {
             }
         });
     }
+
+    async findTransfers(purseUref) {
+        return this.models.Transfer.findAll({
+            where: {
+                [Op.or]: [
+                    {
+                        sourcePurse: purseUref
```

Comment:
There is no index for sourcePurse and targePurse, the find-all method could be very slow.

Reply:
done